lib/post.php

```diff
@@ -331,14 +331,14 @@ public function meta() {
 			'published'    => 'publish' === $this->status() ? true : false,
 		);
 
-		//convert traditional post_meta values, hide hidden values
+		//convert traditional post_meta values, hide hidden values, skip already populated values
 		foreach ( get_post_custom( $this->id ) as $key => $value ) {
 
-			if ( '_' === substr( $key, 0, 1 ) ) {
+			if ( '_' === substr( $key, 0, 1 ) || isset($meta[$key]) ) {
 				continue;
 			}
 
-			$meta[ $key ] = $value;
+			$meta[$key] = $value;
```

Comment:
WordPress coding standards require the space, please don't delete them. Check the Scrutinzr output for other coding style checks.